src/corehost/build.sh

```diff
@@ -18,7 +18,7 @@ init_distro_name_and_rid()
         export __rid_plat=rhel.7
     elif [ "$(cat /etc/*-release | grep -cim1 debian)" -eq 1 ]; then
         export __distro_name=debian
-        export __rid_plat=
+        export __rid_plat=debian.8
     else
```

Comment:
@schellap, I'm not 100% clear on what you are asking.  I verified a runtime.debian.8-x64.Microsoft.NETCore.DotNetHostPolicy.1.0.1-rc2-002365-00.nupkg package is being produced which contains a runtimes\debian.8-x64\native\libhostpolicy.so.